src/index.html

```diff
@@ -5,13 +5,151 @@
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <p>HTML Form</p>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="fieldset_block">
+        <legend>Personal information</legend>
+        <label for="surmane">Surmane:</label>
+        <input
+          class="form_field"
+          type="text"
+          id="surmane"
+          name="sur"
+          autocomplete="off"
+          >
+        <br>
+        <label for="name">Name:</label>
+        <input 
+          class="form_field"
+          type="text"
+          id="name"
+          name="name"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
        <input 
          class="form_field"
          type="text"
          id="name"
          name="name"
          autocomplete="off"
        >
```
Закрывающую скобку выравнивай с открывающей